### ISSUE TITLE
[BUG]: crossorigin attribute is not supported

### ISSUE THREAD

**Issue by StevenFrostwick** [paypal/react-paypal-js#421]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### 🐞 Describe the Bug

The `crossorigin` attribute cannot be set in `react-paypal-js`. The `paypal-js` package does support this though, so you need to upgrade this package to use the latest version of `paypal-js`.

### 🔬 Minimal Reproduction

```markdown
1. Import the `PayPalScriptProvider`
2. Add `crossorigin` as an option e.g.


<PayPalScriptProvider
  deferLoading={
    !paymentSectionData.allowedPayments.includes(
      PaymentMethods.PAYPAL_BNPL
    )
  }
  options={{
    crossorigin: 'anonymous'
}}>


Note that the PayPal script tag added to the `<head>` of the document has the `crossorigin` added as a query parameter. It should not be added as a query parameter - it should be added as an attribute to the script tag.
```

### 🌍 Environment

```markdown
| Software         | Version(s) |
| ---------------- | ---------- |
| react-paypal-js  |            |
| Browser          |            |
| Operating System |            |
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### ➕ Anything else?

This has been fixed in the `paypal-js` package: https://github.com/paypal/paypal-js/commit/d504cc755b497616aff455d0c376378cda3ae3e8

**Comment by StevenFrostwick** [comment]
@gregjopa - does your team still maintain this library?

**Comment by gregjopa** [comment]
Hi @StevenFrostwick sorry for the delay. PR #422 will upgrade this library to the latest version of paypal-js.

**Comment by StevenFrostwick** [comment]
Awesome, thanks Greg!

**Comment by gregjopa** [comment]
Hi @StevenFrostwick, we just published [8.1.4](https://www.npmjs.com/package/@paypal/react-paypal-js/v/8.1.4). In 8.1.4 react-paypal-js should have the latest version of paypal-js.

I'm going to close this out. If you are still have issues, let us know and we can reopen it.

**Comment by StevenFrostwick** [comment]
Thanks everyone!
